packages/solid/components/Column/Column.tsx

```diff
@@ -27,7 +26,17 @@ import type { ColumnProps } from './Column.types.js';
 const Column: Component<ColumnProps> = (props: ColumnProps) => {
   const onUp = handleNavigation('up');
   const onDown = handleNavigation('down');
-  const scroll = createMemo(() => withScrolling(false, props.y));
+  const scroll = createMemo(() => {
+    let y;
+    if (props.style) {
+      if (Array.isArray(props.style)) {
+        y = props.style[0].y || 0;
+      } else {
+        y = props.style.y || 0;
+      }
+    }
+    return withScrolling(false, props.y || y);
```

Comment:
in an instance where the props.style[0].y is undefined but props.style[1].y is, this will set y to 0 instead of the correct value. is there a way we can write this to be a little more resilient?

Reply:
I think this is a bit better than what we had before - but at some point there should be a resolvedStyle or ability to just get the Y value from the node. Didn't want to refactor this to that point yet...

Reply:
I need more time to ponder it - I don't think that top one will really happen. TS is complaining cause style can be an array. But most of the time it won't be - we just need it for components... 

Ugh now I think I should change this around to use onCreate. 🤣

Reply:
I added one more for the array check. Like I said I don't think it will happen... Eventually may remove that...

Reply:
yeah this is fine for now